cookbooks/aws-parallelcluster-scheduler-plugin/recipes/update_head_node.rb

```diff
@@ -15,6 +15,13 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+ruby_block "update_shared_storages" do
+  block do
+    run_context.include_recipe 'aws-parallelcluster-config::update_shared_storages'
```

Comment:
I need some context on doing this, cannot we lazy evaluate the variable inside update_shared_storages?